third-culture-software/bhima
# Allow 0 for unit prices in patient invoices

Updates the patient invoice form to allow items with unit price 0.
Note that there are some legitimate reasons why one might want a unit price for a service (or product) provided to a patient to be zero.  This ensures that is possible.

Closes https://github.com/IMA-WorldHealth/bhima/issues/7518

*TESTING*
- Use bhima_test
- Start with `master`
- Go to `Finance > Patient Invoice`
   - Select a patient
   - Add an item (if one is not already shown)
   - Chose a diagnostic test such as: DDGT_GLUC5K-_0  Glucotest, kit 500nm, Unité
   - Set the unit price to 0
   - Notice that you cannot save the invoice because the unit price cannot be zero.
- Switch to a branch for this pull request
- Restart the node server:  `npm run dev`
- Retry the procedure above.  It should now work.
- Once the patient invoice is completed, you can see that invoice in the `Finance > Invoice Registry` page.  Also, an Invoice Registry report will show the new invoice correctly.
